DISCUSSION.md

```diff
@@ -13,7 +13,7 @@ this branch:
 - [ ] Extract anchors into `_content.json` via `DirectoryTreePlugin`
 - [ ] Finish re-incorporating mobile sidebar
 - [ ] Re-integrate google-analytics
-- [ ] Re-incorporate `redirects.json`
+- [x] Re-incorporate `redirects.json` (Eugene)
```

Comment:
Can you drop this change to fix the conflict? I removed `DISCUSSION.md` in favor of github threads (i.e. PRs and issues). It wasn't meant to be a long-lived document anyway, just gave some perspective on various issues.